themes/mlibrary_new/custom.php

```diff
@@ -103,24 +103,23 @@ function mlibrary_new_display_exhibit_type_of_item($rawAttachment)
 {
    $itemType = 'Still Image';
 
-   if ($rawAttachment[0]->getItem()->getItemType()!=null) {
-       $itemType = $rawAttachment[0]->getItem()->getItemType()->name;
+   if ($rawAttachment->getItem()->getItemType()!=null) {
+       $itemType = $rawAttachment->getItem()->getItemType()->name;
  }
 
    return $itemType;
```

Comment:
These could be simplified by making this a guardian statement:
```
if (empty($itemType = $rawAttachment->getItem()->getItemType())) {
  return 'Still Image';
}
return $itemType->name;
```

Reply:
Agree, will update it.